src/lib/y2storage/blk_device.rb

```diff
@@ -82,6 +82,36 @@ def blk_filesystem
       encrypted? ? encryption.direct_blk_filesystem : direct_blk_filesystem
     end
 
+    alias_method :filesystem, :blk_filesystem
+
+    # Label of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_label
+      return nil unless blk_filesystem
+      blk_filesystem.label
+    end
+
+    # UUID of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_uuid
+      return nil unless blk_filesystem
+      blk_filesystem.uuid
+    end
+
+    # Type of the filesystem, if any
+    # @return [Filesystems::Type, nil]
+    def filesystem_type
+      return nil unless blk_filesystem
+      blk_filesystem.type
+    end
+
+    # Mount point of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_mountpoint
+      return nil unless blk_filesystem
+      blk_filesystem.mountpoint
+    end
```

Comment:
well, I feel a bit wrong about these methods. Its name and prefix `filesystem_` indicate that it is in fact filesystem methods. If it is planned as shortcut, then please remember that target is SLE13 and I hope there will be new ruby so it can be called as `filesystem&.mountpoint`